src/account.c

```diff
@@ -1774,6 +1774,8 @@ int account_uri_complete(const struct account *acc, struct mbuf *buf,
 
 	if (0 != re_regex(uri, len, "[^@]+@[^]+", NULL, NULL) &&
 		1 != uri_is_ip) {
+		if (acc->regint == 0)
+			return -1;
```

Comment:
A meaningful error code should be used, like `ENOTSUP`

Reply:
Sorry, this was a miss-understanding.

We should return 0 in this case. So the call to a local hostname should be possible.